components/ws-daemon/pkg/content/service.go

```diff
@@ -282,8 +282,10 @@ func getCheckoutLocation(req *api.InitWorkspaceRequest) string {
 		}
 	}
 	if ir, ok := spec.(*csapi.WorkspaceInitializer_Prebuild); ok {
-		if ir.Prebuild != nil && ir.Prebuild.Git != nil {
-			return ir.Prebuild.Git.CheckoutLocation
+		if ir.Prebuild != nil {
+			if ir.Prebuild.GetGit() != nil {
```

Comment:
This does not prevent an out-of-bounds panic. The Git array not be nil, yet be empty. Better:

```
if len(ir.Prebuild.Git) > 0 {
    ...
}
```